eng/pipelines/common/global-build-job.yml

```diff
@@ -222,10 +224,18 @@ jobs:
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-wasmer.cmd $(Build.SourcesDirectory)\wasm-tools
         displayName: Install wasmer
 
-    - ${{ if and(eq(parameters.osGroup, 'windows'), eq(parameters.archType, 'x64')) }}:
-      # Install LLVM for the win-x64 build as it will build the clrjit for browser_wasm cross compilation
-      - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.cmd $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
-        displayName: Install/build LLVM
+    - ${{ if eq(parameters.archType, 'x64') }}:
+      - ${{ if eq(parameters.osGroup, 'windows') }}:
+        - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/set-cmake-path.cmd
+          displayName: Set CMake path
+        - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 -CI -InstallDir $(Build.SourcesDirectory)/wasm-tools -Configs ${{ parameters.buildConfig }}
+          displayName: Install/build LLVM
+      - ${{ if eq(parameters.osGroup, 'linux') }}:
+        - script: $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-pwsh.sh $(Build.SourcesDirectory)/wasm-tools
+          displayName: Install Powershell 7
+        # Install LLVM for the win-x64 and linux-x64 build as it will build the clrjit for browser_wasm cross compilation
+        - script: $(Build.SourcesDirectory)/wasm-tools/powershell7/pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 -CI -InstallDir $(Build.SourcesDirectory)/wasm-tools -Configs ${{ parameters.buildConfig }}
+          displayName: Install/build LLVM
```

Comment:
This will install tools for host configurations (`windows_x64`, `linux_x64`) that do not actually target WASM. I think the LLVM stuff should be deleted for `windows_x64` and `Install Powershell 7` folded into the above block (under `{{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm')) }}:`) under `if ne(parameters.hostedOs, 'windows')` (this way it will, hopefully, also work for macOS if/when that comes).